src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -112,6 +115,14 @@ public boolean hasReachedRenewalLimit(Loan loan) {
     return reachedNumberOfRenewalsLimit(loan) && !unlimitedRenewals();
   }
 
+  public boolean hasAllowRecallsToExtendOverdueLoans() {
```

Comment:
I think that `allow recalls to extend overdue loans` makes sense without the `has` prefix. What is the intent of that prefix? 

```suggestion
  public boolean allowRecallsToExtendOverdueLoans() {
```

Reply:
This has been updated.